src/optimizer/agg_key_dependency_optimizer.cpp

```diff
@@ -67,16 +67,17 @@ AggKeyDependencyOptimizer::resolveKeysAndDependentKeys(const binder::expression_
     for (auto& expression : keys) {
         if (expression->expressionType == PROPERTY) {
             auto propertyExpression = (binder::PropertyExpression*)expression.get();
-            if (propertyExpression->isPrimaryKey() || propertyExpression->isInternalID()) {
+            if (propertyExpression->isPrimaryKey() ||
```

Comment:
Does this imply we should try to collapse this if-else statement?

Reply:
Uh, I don't understand the question. `clang-tidy` emits a warning here because branches 1 & 3 have identical bodies. However, rewriting this logic makes it just wrong, so I've instead told clang-tidy to ignore this check for this line.